backend/src/main/java/atlantbh/restaurants/services/MenuService.java

```diff
@@ -0,0 +1,16 @@
+package atlantbh.restaurants.services;
+
+import atlantbh.restaurants.models.Menu;
+import atlantbh.restaurants.models.filters.MenuFilterBuilder;
+import atlantbh.restaurants.models.sortkeys.MenuSortKeys;
+import atlantbh.restaurants.repositories.MenuRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MenuService extends  BaseService<Menu, MenuSortKeys, MenuFilterBuilder, MenuRepository> {
+    public List<Menu> getByRestaurant(Long restaurant) {
```

Comment:
filter

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb